aptos-move/framework/aptos-framework/sources/reconfiguration.spec.move

```diff
@@ -1,10 +1,65 @@
 spec aptos_framework::reconfiguration {
     spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
+
         // After genesis, `Configuration` exists.
         invariant [suspendable] chain_status::is_operating() ==> exists<Configuration>(@aptos_framework);
         invariant [suspendable] chain_status::is_operating() ==>
             (timestamp::spec_now_microseconds() >= last_reconfiguration_time());
     }
+    
+    /// A schem to ensure caller is admin
+    spec schema Init {
+        aptos_framework: &signer;
+
+        let addr = signer::address_of(aptos_framework);
+        aborts_if !system_addresses::is_aptos_framework_address(addr);
+    }
+
+    /// requires Account and Configuration 
```

Comment:
Can you capitalize the first letter, and also elaborate this sentence?

Reply:
The revision is complete and the comments have been reworked.

Reply:
Done and the comments have been reworked.